src/main/java/seedu/diary/logic/commands/interviewsubcommands/InterviewDeleteCommand.java

```diff
@@ -43,7 +43,7 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         Interview interviewToDelete = interviews.get(interviewIndex.getZeroBased());
-        interviews.remove(interviewIndex.getZeroBased());
+        internshipToModify.deleteInterview(interviewToDelete);
         model.displayInternshipDetail(internshipToModify);
```

Comment:
can just remove this

Reply:
As in remove model.displayInternshipDetail(internshipToModify);

Reply:
Hmm I kept it there so that whenever the Interview commands are called, the IA details will be displayed. So the user can see the changes. I think its good that you can see the interview added/deleted/edited immediately after you called the command?